src/GeneralStateTestsFiller/stTransactionTest/CreateTransactionEOF1Filler.yml

```diff
@@ -0,0 +1,141 @@
+# Creation transaction deploying code with 0xef first byte (EIP-3541)
+---
+CreateTransactionEOF1:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 100000
+      code: ''
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data:
+      #- ':yul { mstore8(0, 0xef) return(0, 1) }'
+      - ':label deploying_0xef :raw 0x60ef60005360016000f3'
```

Comment:
Yul didn't  work?

Reply:
It did, I just wanted to show raw code here, maybe include it in EIP text.